collector/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ project(collector-bin)
 find_package(Threads)
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR})
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC -Wall --std=c++17 -pthread -Wno-deprecated-declarations")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC -Wall --std=c++17 -pthread -Wno-deprecated-declarations -fno-omit-frame-pointer -rdynamic")
```

Comment:
What are the implications of always compiling with these two flags? Is the final binary bigger? Are symbols preserved in it? Does it affect performance in any meaningful way? I'll do some research on my own, but would love to hear your thoughts.

Reply:
Yes, me reading the PR description 🤦🏻‍♂️